index.html

```diff
@@ -8,17 +8,22 @@
       // See https://github.com/w3c/respec/wiki/ for how to configure ReSpec
       var respecConfig = {
         specStatus: "CG-DRAFT",
-        shortName: "beforeinstallprompt",
         editors: [{
           name: "Marcos Cáceres"
```

Comment:
Change to yourself. (Since this was empty before, I think @marcoscaceres won't mind not being an editor; we can always add back later.)